docs/tutorial/image_det.md

```diff
@@ -1,3 +1,166 @@
+image size (integer x integer):  128x128
+how many epochs do you run training (integer):  100
+initial learning rate:  0.001
+choose learning rate setting(tune1 / tune2 / tune3 / fixed):  tune1 -> "2 times decay": tune1
+apply quantization at the first layer?  yes
+```
+
+## Train a neural network
+
+Train your model by running `blueoil train` command with model configuration. 
+
+    $ ./blueoil.sh train config/{Model name}.yml
+
+When training is started, training log and checkpoints are generated under `./saved/{Mode name}_{TIMESTAMP}` directory.
+
+Training is running on TensorFlow backend. So you can use TensorBoard to visualize your training process. 
+
+    $ ./blueoil.sh tensorboard saved/{Model name}_{TIMESTAMP} {Port}
+
+## Convert training result to FPGA ready format.
+
+Convert trained model to executable binary files for x86, ARM, and FPGA.
+Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
+
+    $ ./blueoil.sh convert config/[Model name].yml saved/{Mode name}_{TIMESTAMP}
+
+`Blueoil convert` automatically executes some conversion processes.
+- Convert Tensorflow checkpoint to protocol buffer graph.
+- Optimize graph
+- Generate source code for executable binary
+- Compile for x86, ARM and FPGA
+
+If conversion is successful, output files are generated under `./saved/{Mode name}_{TIMESTAMP}/export/save.ckpt-{Checkpoint No.}/{Image size}/output`.
+
+```
+output
+ ├── fpga (include preloader and FPGA configuration file)
+ │   ├── preloader-mkpimage.bin
+ │   └── soc_system.rbf
+ ├── models
+ │   ├── lib (include trained model library)
+ │   │   ├── lib_arm.so
+ │   │   ├── lib_fpga.so
+ │   │   └── lib_x86.so
+ │   └── meta.yaml (model configuration)
+ ├── python
+ │   ├── lmnet (include pre-process/post-process)
+ │   ├── README.md
+ │   ├── requirements.txt
+ │   ├── run.py (inference script)
+ │   └── usb_camera_demo.py (demo script for object detection)
+ └── README.md
+```
+
+## Run inference script on x86 Linux (Ubuntu 16.04)
+
+- Prepare inference images (not included in the training dataset)
+
+	You can find test imgaes on [Creative Commons](https://ccsearch.creativecommons.org/). [Sample](https://ccsearch.creativecommons.org/photos/ddfd33a6-140f-49a3-85b8-3bf58a877990)
+
+- Run inference script
+
+    Explore into the `output/python` directory, and
+    run `run.py` and inference result is saved in `./output/output.json`.
+
+    Note: If you run the script for the first time, you have to setup a python environment (2.7 or 3.5+) and install requirements python packages.
+
+	```
+	$ cd {output/python directory}
+	$ sudo pip install -r requirements.txt  # only the first time
+	$ python run.py \
+	      -i {inference image path} \
+	      -l ../models/lib/lib_x86.so \
+	      -c ../models/meta.yaml
+	```
+
+	*Tips:* The default thredhold for object detection is `0.05`. If you find too many boxses when running demo, you can edit `meta.yml` and set threshold about `0.4` or `0.5` as below code.
```

Comment:
boxses -> boxes

I think it maybe better to use
"set threshold to 0.4 or 0.5" if you want the uses to set the threshold to either 0.4 tor 0.5 or
"set threshold between 0.4 and 0.5" if any number in the range is okay.